chips/veer_el2/src/chip.rs

```diff
@@ -101,12 +106,12 @@ impl<'a, I: InterruptService + 'a> SweRVolf<'a, I> {
     }
 }
 
-impl<'a, I: InterruptService + 'a> kernel::platform::chip::Chip for SweRVolf<'a, I> {
-    type MPU = ();
+impl<'a, I: InterruptService + 'a> kernel::platform::chip::Chip for VeeR<'a, I> {
+    type MPU = PMPUserMPU<4, SimplePMP<8>>;
```

Comment:
Why not use ePMP?

Reply:
Without weighing in on whether this should use ePMP by default, here's an example ePMP instantiation for the QEMU rv32i virt chip:
https://github.com/tock/tock/blob/d0d77236992cb4b5024fc9a9c93ede99c08e5ba0/chips/qemu_rv32_virt_chip/src/chip.rs#L25-L34

This chip's board file contains the corresponding constructor invocation.

Reply:
I've used SimplePMP as it was enough to show that a userspace application won't be able to overwrite the kernel code. I can see how we can get better protection with ePMP though.

I am willing to replace the configuration but will need another moment for this (as it seemed not to work out of the box for me). The link provided will surely be very useful!

Reply:
I think `SimplePMP` is totally fine, and we can always switch to an ePMP configuration later. Any reasonable memory protection support meets the bar for upstreaming a board.